platform-includes/getting-started-install/javascript.nextjs.mdx

```diff
@@ -1,24 +1,25 @@
-We recommend installing the SDK through our installation wizard:
+To install Sentry using the installation wizard, run the following command within your project:
 
 ```bash
 npx @sentry/wizard@latest -i nextjs
 ```
 
-The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:
+The wizard starts guiding you through the setup process, which also includes enabling the Sentry features listed above.
```

Comment:
We would need to remove the part about features if we remove the features section above.